File: sdks/go/models/environment_variable_out.go
package models

import "time"

type EnvironmentVariableOut struct {
	Id            string    `json:"id"`
	EnvironmentId string    `json:"environmentId"`
	Key           string    `json:"key"`
	Value         string    `json:"value"`
	IsSecret      bool      `json:"isSecret"`
	CreatedAt     time.Time `json:"createdAt"`
	UpdatedAt     time.Time `json:"updatedAt"`
}

File: sdks/java/lib/src/main/java/com/hooksniff/models/EnvironmentVariableOut.java
package com.hooksniff.models;

import com.fasterxml.jackson.annotation.JsonProperty;

public class EnvironmentVariableOut {
    public String id;
    @JsonProperty("environment_id") public String environmentId;
    public String key;
    public String value;
    @JsonProperty("is_secret") public boolean isSecret;
    @JsonProperty("created_at") public String createdAt;
    @JsonProperty("updated_at") public String updatedAt;
}
